notesserver/highlight.py

```diff
@@ -39,13 +39,28 @@ def build_search_kwargs(self, *args, **kwargs):
             highlight_options = {
                 'fields': {
                     content_field: {'store': 'yes'},
-                }
+                    'tags': {'store': 'yes'},
+                },
             }
             if isinstance(highlight, dict):
                 highlight_options.update(highlight)
             res['highlight'] = highlight_options
         return res
 
+    def _process_results(
```

Comment:
@jimabramson Wouldn't moving highlighting logic into the API handlers require them to handle many of the concerns that ES currently deals with, like tokenizing and stemming of searches? The user could potentially search for both a tag and text in the same query. This approach could end up being messier than the current one.